test/expected/bgw_launcher.out

```diff
@@ -0,0 +1,347 @@
+\c single_2 :ROLE_SUPERUSER
+/*
+ * Note on testing: need a couple wrappers that pg_sleep in a loop to wait for changes
+ * to appear in pg_stat_activity.
+ * Further Note: PG 9.6 changed what appeared in pg_stat_activity, so the launcher doesn't actually show up. 
+ * we can still test its interactions with its children, but can't test some of the things specific to the launcher. 
+ * So we've added some bits about the version number as needed. 
+ */
+CREATE FUNCTION wait_worker_counts(INTEGER, INTEGER, INTEGER) RETURNS BOOLEAN LANGUAGE PLPGSQL AS
```

Comment:
can you use named parameters? It is pretty hard to understand what these parameters are. What does a `true` or `false` return value signify?